### DISCUSSION TITLE
how to configure protobuf in kafka-ui -- looking for some guidance

### DISCUSSION THREAD

**Discussion by priyavj08** [provectus/kafka-ui#1178] [Q&A] [answered]
hi,
In our kafka cluster, protobuf encoded messages are stored in topics. How to setup the kafka-ui with protobuf so the messages can be decoded? 
Is there a way to locally set it up as we don't have support for schema registry?  I couldn't find any examples

looking for some pointers

appreciate any help

thanks
Priya

**Comment by Haarolean** [comment, marked as answer]
Its possible to set it up via protobuf files:
https://github.com/provectus/kafka-ui/blob/master/kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/ClustersProperties.java#L32
#849 
#1699 (more documentation expected)
